documentation/orgchart.md

```diff
@@ -0,0 +1,14 @@
+DevOps OrgChart
+
+| Role | SlackName |
+| --- | --- |
+| Main Point of Contact | **@dmangot** |
+| AWS Team | @skuenzli, @imonthercks, @cmac |
+| Azure Team | @Troy Witthoeft, @Jonathan Vermeij |
+| GCP Team | TBD |
+| Auth Team | **@imonthercks** |
+| Airtable Support | **@psprings**, @ken |
+| Github Support | @Frank Prins and @0x2b3bfa0 |
+
+
+_**@username** denotes primary point of contact for that team_
```

Comment:
Does this organization chart replace the _directory_ section on [/README.md](/README.md)?